app/src/main/java/com/example/nvdovin/weatherapp/utils/TemperatureScales.java

```diff
@@ -0,0 +1,7 @@
+package com.example.nvdovin.weatherapp.utils;
+
+public class TemperatureScales {
+    public static int getCelsiusScale(){
+        return 273;
```

Comment:
what a hell is this? why do you use method to return int value? by the context it should be a constant, and  be accessed   through classname.CONSTANT_NAME